udf/mrn_udf_highlight_html.cpp

```diff
@@ -46,13 +47,20 @@ typedef struct st_mrn_highlight_html_info
     bool used;
     grn_obj *table;
     grn_obj *default_column;
+    const char* query;
+    unsigned long query_length;
   } query_mode;
+  struct {
+    bool used;
+    const char *open_tag;
+    const char *close_tag;
+  } specify_tag;
```

Comment:
How about defining them in top-level and use them for the default tags (`<span class="keyword">` and `</span>`) too?

```suggestion
  const char *open_tag;
  const char *close_tag;
```

Reply:
Fixed.